index.js

```diff
@@ -15,7 +15,6 @@
 var createError = require('http-errors')
 var debug = require('debug')('send')
 var deprecate = require('depd')('send')
-var destroy = require('destroy')
```

Comment:
Or I guess it's no longer relevant since I think the changes here are to manually manage the `fd` instead of letting Node.js's read stream do so?